src/main/java/se/sundsvall/snailmail/service/SnailMailService.java

```diff
@@ -63,6 +67,21 @@ public void sendSnailMail(final String municipalityId, final SendSnailMailReques
 		requestRepository.save(Mapper.toRequest(request, citizen, department));
 	}
 
+	private void validateCitizenAddress(CitizenExtended citizen) {
+		//Verify that we have name, lastname, address, postal code and city, otherwise we cannot send it as a snail mail.
+		Optional.ofNullable(Mapper.toRecipient(citizen))
+				.filter(recipientEntity -> isNotBlank(recipientEntity.getGivenName()))
+				.filter(recipientEntity -> isNotBlank(recipientEntity.getLastName()))
+				.filter(recipientEntity -> isNotBlank(recipientEntity.getAddress()))
+				.filter(recipientEntity -> isNotBlank(recipientEntity.getPostalCode()))
+				.filter(recipientEntity -> isNotBlank(recipientEntity.getCity()))
+			.orElseThrow(() -> Problem.builder()
+				.withTitle("Incomplete recipient address information")
+				.withStatus(INTERNAL_SERVER_ERROR)
+				.withDetail("Missing required fields in the recipient address")
```

Comment:
A client is probably also wondering which fields are missing, not just that some required fields are missing?

Maybe: 
`Citizen with partyId: "xxx" does not contain required address fields: address, postalCode, city.`